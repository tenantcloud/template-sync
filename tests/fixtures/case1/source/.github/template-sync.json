{
	"repositories": [
		{
			"url": "template",
			"branch": "master",

			"after": {
				"plugins": [
					".github/template-sync/override.js"
				]
			}
		},
		{
			"url": "other-template",
			"branch": "alpha"
		}
	]
}
