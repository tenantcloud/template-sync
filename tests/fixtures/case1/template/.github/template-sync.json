{
	"repositories": [
		{
			"url": "template",
			"branch": "changed-branch"
		},
		{
			"url": "added-template",
			"branch": "master"
		}
	]
}
