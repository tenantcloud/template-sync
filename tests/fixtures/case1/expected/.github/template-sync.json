{
	"repositories": [
		{
			"url": "template",
			"branch": "changed-branch",

			"after": {
				"plugins": [
					".github/template-sync/override.js"
				]
			}
		},
		{
			"url": "added-template",
			"branch": "master"
		},
		{
			"url": "other-template",
			"branch": "alpha"
		}
	]
}
